systemtests/go.mod

```diff
@@ -0,0 +1,171 @@
+module cosmossdk.io/systemtests
```

Comment:
Could you submit a PR in https://github.com/cosmos/vanity changing https://github.com/cosmos/vanity/blob/main/vangen.json#L24 and re-generating before this gets merged?